lib/trento_web/views/v1/sap_system_view.ex

```diff
@@ -17,11 +17,38 @@ defmodule TrentoWeb.V1.SapSystemView do
     database
     |> Map.from_struct()
     |> Map.delete(:__meta__)
+    |> Map.delete(:deregistered_at)
     |> Map.put(:database_instances, rendered_database_instances)
     |> add_system_replication_status_to_secondary_instance
   end
 
   def render("database_instance.json", %{instance: instance}) do
+    instance
+    |> Map.from_struct()
+    |> Map.delete(:__meta__)
+    |> Map.delete(:absent_at)
+    |> Map.delete(:host)
+  end
+
+  def render("broadcast_database.json", %{
+        database:
```

Comment:
I think we could render inside this the `database.json` view.
After all, they have the exact same fields, both the database and instance (removing the `deregistered_at` is not a big deal, we don't use it in the frontend).
Anyway, if having it explicit makes sense to you, all good from my side, but they should have the same content